resolwe_bio/processes/support_processors/archive.yml

```diff
@@ -6,10 +6,11 @@
     executor:
       docker:
         image: "resolwebio/archive-samples"
-  version: 0.0.2
+  version: 0.0.3
   type: "data:archive:samples"
   category: other
   persistence: TEMP
+  scheduling_class: interactive
```

Comment:
@kostko We need this for the [archiver process](https://github.com/genialis/resolwe/blob/master/resolwe/toolkit/processes/archiver.yml) in resolwe too.